docs/application/toc_all.md

```diff
@@ -56,6 +56,7 @@
 ##### [Overview](/application/dotnet/guides/applications/ui-app.md)
 ##### [Watch Application](/application/dotnet/guides/applications/watch-app.md)
 ##### [Widget Application](/application/dotnet/guides/applications/widget-app.md)
+##### [Component Based Application](/application/dotnet/guides/applications/component-based-app.md)
```

Comment:
Also a question. Do you think this pages doesn't needed to be added in dotnet/guides/applications/overview.md?

Reply:
@upple, that is valid point, could you please update that file also, since other UI applications are listed there.

Reply:
I added description for component based app to overview.md